evaluation/benchmarks/EDA/scripts/run_infer.sh

```diff
@@ -36,10 +36,10 @@ fi
 
 # IMPORTANT: Because Agent's prompt changes fairly often in the rapidly evolving codebase of OpenHands
 # We need to track the version of Agent in the evaluation to make sure results are comparable
-AGENT_VERSION=v$(poetry run python -c "import openhands.agenthub; from openhands.controller.agent import Agent; print(Agent.get_cls('$AGENT').VERSION)")
+OPENHANDS_VERSION=v$(poetry run python -c "import openhands.agenthub; from openhands.controller.agent import Agent; print(Agent.get_cls('$AGENT').VERSION)")
```

Comment:
Delete this in favor of `get_openhands_version`?

Reply:
Good catch! Deleted in the latest commit.